fall18/p4/README.md

```diff
@@ -1,3 +1,308 @@
 # Project 4
 
-More details coming soon.
+This project is for all you movie enthusiasts out there! This week, we will be looking at data set about ~1,000 most popular movies on IMDB in the last 10 years. Each movie in the dataset has like cast, director, rating, genre, runtime, budget, etc.
+
+We will be working with **strings**, **loops**, **conditionals** and **command line arguments** in this project.
+
+To start, download the 4 files given below into your project directory.
+
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/main.py)
+* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/project.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/test.py)
+* [IMDB-Movie-Data.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/IMDB-Movie-Data.csv)
+
+Open up the IMDB-Movie-Data.csv file and get familiar with it! Notice how the movies are indexed starting from zero, how the 'genres' and 'cast' are listed as several actors separated by spaces. There's something strange about the column 'budget', can you tell what it is?
+
+**A reminder:** You should make your changes ONLY to the **main.py** file.
+DO NOT EDIT ANY OTHER FILES.
+
+After downloading all the files into the same directory try running the command below:
+
+```
+python test.py
+```
+
+and you should see the following: 
+```
+To be filled
```

Comment:
Your Python version: Python 3.6.6

Running your program with this command: python3 main.py

RESULTS:
  test_1: PASS
  test_2: missing function reverseString
  test_3: missing function checkPalindrome
  test_4: missing function checkPalindrome
  test_5: missing function findPalindromeMovie
  test_6: missing function encodeString
  test_7: missing function encodeString
  test_8: missing function countMoviesByDirector
  test_9: missing function findNumSequels
  test_10: missing function mainActor
  test_11: missing function countMoviesByActor
  test_12: missing function findHighestRevenue
Score: 8%